packages/php-db-import-export/src/Backend/Assert.php

```diff
@@ -24,6 +31,7 @@ public static function assertSameColumns(
         array $ignoreSourceColumns = [],
         array $simpleLengthTypes = [],
         array $complexLengthTypes = [],
+        int $assertOptions = self::ASSERT_LENGTH,
```

Comment:
chtěl sem aby to bylo trochu univerzálnější a dalo sa tam kdyžtak přidat víc options a vyřadit nějaké asserty